src/creator.coffee

```diff
@@ -94,6 +96,7 @@ SurveyWidget.controller 'SurveyWidgetController', [ '$scope','$mdToast','$mdDial
 		$scope.$apply ->
 			$scope.title = title
 			$scope.groups = qset.options.groups
+			$scope.OneQuestionAtATime = qset.options.OneQuestionAtATime
```

Comment:
We need to ensure the creator and player are backwards compatible with previously created qsets which will not have this property. There should be a check here to ensure that if the `OneQuestionAtATime` value is not present, it should be handled appropriately by being set to false.